server/flaskr/database/handlers/notification_job_data_handler.py

```
@@ -0,0 +1,70 @@
+            "SET orgId = %s, targetId = %s, type = %s, payload = %s, scheduledAt = %s "
+            "WHERE id = %s;"
+        )
+        params = (job.orgId, job.targetId, job.type, job.payload, job.scheduledAt, jobId)
+        try:
+            write_query(query, params)
+            return True
+        except Exception:
+            return False
+
+    @classmethod
+    def mark_as_sent(cls, jobId: int, sentAt: datetime = None) -> bool:
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)